client/webserver/site/package.json

```diff
@@ -23,7 +23,7 @@
     "bootstrap": "^4.6.0",
     "clean-webpack-plugin": "^3.0.0",
     "css-loader": "^5.1.2",
-    "css-minimizer-webpack-plugin": "^1.2.0",
+    "css-minimizer-webpack-plugin": "^3.0.1",
```

Comment:
The CI check failed due to a vulnerability found in this package. It is now passing again after updating this.

Reply:
I don't think CI failed because of a vuln in this. https://github.com/decred/dcrdex/runs/2869810692#step:4:6
Appears to be some transient npm error.

Rather than have a  +7,140 −10,057 diff, pls revers this last diff and we'll address it in another PR.
I have a branch with a batch of npm dep updates.